### PR TITLE
CircleCI Update Chrome to latest before we start our tests.  Only updates Webdriver if needed

### PR DESCRIPTION
**Pull request by shermaneric**

### CODE REVIEW

File: gruntfile.js [hunk 1]
```diff
@@ -9,6 +9,14 @@ if (ab.enableTransform(module)) {
 module.exports = function(grunt) {
   require('load-grunt-tasks')(grunt);
 
+  grunt.registerTask('webdriverUpdateWithPinnedChromedriver', function() {
+    //ensures correct pinned ChromeDriver in config.js exists, otherwise fetch it.
+    if (!grunt.file.exists('./node_modules/webdriver-manager/selenium/chromedriver_' + config.chromedriverVersion)) {
+      console.log('Chromedriver ' + config.chromedriverVersion + ' not found. Running shell:webdriverUpdate.');
+      grunt.task.run('shell:webdriverUpdate');
+    }
+  });
```
Comment (shermaneric) [inline gruntfile.js]: I'm somehwat ambivalent about the above.  Options.
1.  **Keep it the same as current develop** Do not do lines 12-18 and just always `shell:webdriverUpdate` with the chromeDriver version in config.js
2. **Leave the above change in**.  This will always check the **local chromedriver** against the chromeDriver version in config.js and update if needed using `webdriverUpdate`
3. **Enhance the above change** to check for **any** webdriver binary that is not matching a version in config.js and update if needed using `webdriverUpdate`.   [I have an experimental branch working here](https://github.com/blackboard/protractor-sync/compare/feature/experiment_from_update_latest_chrome?expand=1)
4.  **Revert pinning and just take the latest webdriver binaries** each time with no versions specified always running `webdriverUpdate`

Thoughts welcome!  I like #3 above myself but I'll go with what folks think.

Thanks!

**Comment by shermaneric** [comment]
Just fyi, this PR is ready for review again for those interested :)

File: gruntfile.js (hunk 1 above)
Comment (tsu-denim) [reply gruntfile.js]: Option 3 looks good! Can you submit a new PR?

Comment (mindywhitsitt) [reply gruntfile.js]: I feel too far removed from this specific problem to have a helpful thought. The only thing I can offer is the rather general statement that protractor-sync as an open-source project is meant to be usable out-of-the-box by a wide variety of people. So my rather painfully obvious advice is to go with anything that:
1) helps it "just work"
2) respects many possible different setups and doesn't do anything that would annoy anyone whose setup is not what protractor-sync expects

In short, I'll trust your judgment! :)

Comment (shermaneric) [reply gruntfile.js]: thanks mindy!

**Review by tsu-denim — DISMISSED** [review]
Looks good, thanks for keeping it current with new Chrome updates!

**Comment by shermaneric** [comment]
Mind all the dust of extra commits.  This PR is ready for re-review with option 3 (pinning the variant webdriver binaries) + updates to latest chrome based on this updated [Circle Ci 2.0 gist](https://gist.github.com/phra/192b6c0ada3f6eaf4029e430430b79fe)

**Review by ftclausen — APPROVED** [review]
